presto-product-tests/bin/run_on_docker.sh

```diff
@@ -207,7 +207,7 @@ fi
 trap terminate INT TERM EXIT
 
 # start external services
-EXTERNAL_SERVICES="hadoop-master mysql postgres cassandra"
+EXTERNAL_SERVICES="hadoop-master mysql postgres cassandra sqlserver"
```

Comment:
Yes, the tests are passing on Travis. I strong believe we should have a way to run only hadoop-master + presto-master + related container, rather than starting all containers while running any given product-tests.